test/cypress/integration/consumer-tools/youth-financial-education/survey-errors.cy.js

```diff
@@ -51,6 +52,7 @@ describe('Youth Financial Education Survey: Errors', () => {
     refreshErrors();
     cy.get('form .m-notification__error li:nth-child(2) a').click();
     cy.wait(1200);
+    cy.get('.survey-reset--link--wrap').isScrolledTo();
     cy.window().then((win) => {
       expect(win.scrollY).greaterThan(1000);
     });
```

Comment:
Now that we're checking to see if a certain element is in view, I wonder if the check for the correct `scrollY` coordinates is redundant? From what I can tell, the intention of the test is "make sure the right part of the page is visible after taking this action"; seems like that would be satisfied by checking that all the right elements pass the `isScrolledTo` assertion. I might be misunderstanding what's actually being checked there, though.

Reply:
Works for me, I think you are right. Changed in https://github.com/cfpb/consumerfinance.gov/pull/7424/commits/597c32f8ade337145d4f10b4b0cc9eab3d0475f2